lumigator/python/mzai/backend/pyproject.toml

```diff
@@ -34,4 +34,4 @@ dev-dependencies = [
 ]
 
 [tool.uv.sources]
-schemas = { path = "../schemas", editable = true }
+lumigator-schemas = { path = "../schemas", editable = true }
```

Comment:
```suggestion
lumigator-schemas = { path = "../lumigator_schemas", editable = true }
```

Reply:
We will document the difference in the top level README between the backend, sdk and schemas directories, and the top level package namespace.